.github/workflows/release.yml

```diff
@@ -235,7 +239,7 @@ jobs:
             aws s3 cp ./target/${{ matrix.cargo_targets }}/optimized-release/iroh-dns-server s3://vorc/iroh-dns-server-${RELEASE_OS}-${RELEASE_ARCH}-${GITHUB_SHA::7} --no-progress
     
         - name: push release latest
-          if: matrix.os != 'windows-latest'
+          if: matrix.os != 'windows-latest' && (github.event.inputs.mark_latest == 'true' || github.event_name == 'push')
```

Comment:
Can we add parentheses?  Whenever I see `a && b || c` I expect bugs...

Reply:
I did 👀